actions/class.ItemPreview.php

```diff
@@ -70,10 +70,10 @@ public function render(){
         
         $uri = base64_decode($codedUri);
         $item = new core_kernel_classes_Resource($uri);
-        if($path == 'index'){
+        if($path === 'index'){
```

Comment:
No, really not. According http://www.phpbench.com/ and other benchmarks  strict comparison in if operator provide slightly better performance without any side affect on logic in current case. If you prefer  not to pollute commits with so tiny changes not related to essence of issue just let me know and i will not do such thing in future.